runtime/runtime/src/congestion_control.rs

```diff
@@ -85,7 +85,6 @@ impl ReceiptSink {
         bandwidth_scheduler_output: Option<BandwidthSchedulerOutput>,
     ) -> Result<Self, StorageError> {
         if let Some(own_congestion_info) = prev_own_congestion_info {
```

Comment:
Change parameter `prev_own_congestion_info` type from `Option<CongestionInfo>` to `CongestionInfo`.